src/test/java/uk/gov/companieshouse/exemptions/AuthenticationFilterTest.java

```diff
@@ -159,4 +159,41 @@ void doFilterInternalKeyWrongPrivileges() throws ServletException, IOException {
 
         verify(filterChain, times(0)).doFilter(request, response);
     }
+
+    @Test
+    @DisplayName("Set ERIC Id type as an invalid value and fail")
+    void doFilterInternalWithInvalidEricIdentityType() throws ServletException, IOException {
+        when(request.getHeader("ERIC-Identity")).thenReturn("SOME-IDENTITY");
+        when(request.getHeader("ERIC-Identity-Type")).thenReturn("notKeyOrOauth2");
+
+        filter.doFilterInternal(request, response, filterChain);
+
+        verify(filterChain, times(0)).doFilter(request, response);
+    }
+
+    @Test
+    @DisplayName("Send invalid key and throw error")
```

Comment:
DisplayName seems to be the same as the next test. Would you like to change it?

Reply:
done